apidoc/Titanium/UI/iPhone/iPhone.yml

```diff
@@ -202,7 +212,10 @@ properties:
                  Titanium.UI.iPhone.StatusBar.GREY, Titanium.UI.iPhone.StatusBar.LIGHT_CONTENT,
                  Titanium.UI.iPhone.StatusBar.OPAQUE_BLACK, Titanium.UI.iPhone.StatusBar.TRANSLUCENT_BLACK ]
     permission: read-only
-
+    deprecated:
+        since: "3.1.3"
+        notes: Use Titanium.UI.Window.statusBarStyle instead please.
```

Comment:
Missing link (see above), remove the "please". Also check that one in all other places.